edgexfoundry/edgex-go
# feat(all): Enable CORS headers for all services

Add configurations and implement middleware function to control CORS headers.

Close #3757







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
Modify configuration to enable CORS headers

```
  [Service.CORSConfiguration]
  EnableCORS = true
```

Call the any of the Service API which enable CORS
Verify the header match the value under Service.CORSConfiguration
